src/db.ts

```diff
@@ -0,0 +1,354 @@
+import _ from 'lodash';
+import { Client } from 'pg';
+import config from './config';
+
+const client = new Client(config.postgreSQL);
+
+// Table Names
+const CONVERSATIONS = 'conversations';
+const PARTICIPANTS = 'participants';
+const SESSIONS = 'sessions';
+const SEGMENTS = 'segments';
+const METRICS = 'metrics';
+
+// Create Queries
+const CREATE_CONVERSATIONS_TABLE_QUERY = `
+CREATE TABLE IF NOT EXISTS ${CONVERSATIONS}(
+    conversation_id                      VARCHAR(36) NOT NULL,
+    conversation_start                   VARCHAR(25) NOT NULL,
+    conversation_end                     VARCHAR(25),
+    media_stats_min_conversation_mos     INT,
+    media_stats_min_conversation_rfactor INT,
+    originating_direction                VARCHAR(8) NOT NULL,
+    division_ids                         VARCHAR(1000),
+    PRIMARY KEY(conversation_id)
+ );
+`;
+
+const CREATE_PARTICIPANTS_TABLE_QUERY = `
+CREATE TABLE IF NOT EXISTS ${PARTICIPANTS}(
+    participant_id        VARCHAR(36) NOT NULL,
+    conversation_id       VARCHAR(36) NOT NULL,
+    participant_name      VARCHAR(255),
```

Comment:
The API doesn't truncate names to 255, does it? For open-ended strings, use `VARCHAR` or `TEXT` without specifying the length. Either is fine, they're apparently equivalent in Postgres. 

https://www.postgresqltutorial.com/postgresql-char-varchar-text/

> If you do not specify the n integer for the VARCHAR data type, it behaves like the TEXT datatype. The performance of the VARCHAR (without the size n) and TEXT are the same.
> 
> The only advantage of specifying the length specifier for the VARCHAR data type is that PostgreSQL will issue an error if you attempt to insert a string that has more than n characters into the VARCHAR(n) column.
> ...
> Different from other database systems, in PostgreSQL, there is no performance difference among three character types.